### ISSUE TITLE
Order new lens for Laser Cutter Tony

### ISSUE THREAD

**Issue by magman2112** [DoESLiverpool/somebody-should#1749]
The focusing lens for Tony has a developing flaw in the main focal area, so we should buy a replacement before this flaw propagates and stops the laser working.

The lens is 19mm in diameter, with a focal length of 50.8mm (2 inches)

I have identified a good quality lens from Cloudray, which is available from Amazon for £28, as following link:

https://www.amazon.co.uk/Cloudray-Engraver-Cutter-50-8mm-inches

I would recommend getting 2 of these lenses, so we can keep one as a spare.

**Comment by DoESsean** [comment]
Could you check that for us please @magman2112, I'm getting a 404 from that link.

**Comment by magman2112** [comment]
Hi Sean, it looks like I shortened the URL too much. Here it is again.

https://www.amazon.co.uk/Cloudray-Engraver-Cutter-50-8mm-inches/dp/B07B8X7T7G

**Comment by DoESsean** [comment]
Great, thanks. I've ordered a couple of them to arrive tomorrow.

**Comment by Sean-anotherone** [comment]
The lens that arrived is an 18mm, the one in the machine is 19mm ( I measured both last thurs, forgot to say).  If it's not returnable I'll try to make a centering ring, but ~0.48mm thick might be a bit of a challenge.

**Comment by magman2112** [comment]
Another issue we found with the new lenses is that they are the wrong focal length for Tony. @Sean-anotherone and I had assumed that it was 50mm, like the other lasers - we were wrong. 

The actual focal length is 63.5 mm (2.5 inches) - I found a suitable replacement on Amazon and ordered it.

https://www.amazon.co.uk/dp/B07MFQHQ4T?ref=ppx_yo2ov_dt_b_product_details&th=1

This has now been fitted by Sean on last nights Maker Night and is cutting well. Sean also found a few burrs in the laser mounting which have also been fixed.

safe to close this issue now.
